pkg/config/schema/fuzz_test.go

```diff
@@ -159,5 +159,8 @@ func fixProtoFuzzer(codecs serializer.CodecFactory) []interface{} {
 		func(t *types.Value, c fuzz.Continue) {
 			*t = types.Value{Kind: &types.Value_StringValue{StringValue: ""}}
 		},
+		func(t *networking.WorkloadGroup, c fuzz.Continue) {
```

Comment:
I think ideally we will just add interface types here rather than skip the whole thing. So for workload group this is probably the oneOf for the health check.

Reply:
so just populate it?

Reply:
I think this can be like 
```
		func(t *networking.ReadinessProbe, c fuzz.Continue) {
			*t = networking.ReadinessProbe{}
		},
```

So we only skip ReadinessProbe and not the whole WorkloadGround

ideally we are actually not just returning empty objects here and are doing the fuzzing logic ourselves